wallet-new/src/Cardano/Wallet/Kernel/Util.hs

```diff
@@ -60,11 +62,21 @@ m `restrictKeys` s = m `Map.intersection` Map.fromSet (const ()) s
   Dealing with OldestFirst/NewestFirst
 -------------------------------------------------------------------------------}
 
+liftOldestFirst' :: Functor m
+                 => (f a -> m (f a))
+                 -> OldestFirst f a -> m (OldestFirst f a)
+liftOldestFirst' f = fmap OldestFirst . f . getOldestFirst
```

Comment:
Wouldn't `liftOldestFirstM` be a slightly better name? Not very picky about this but some folks are really adamant that the `prime` naming should be used only for strict variants 😉

Reply:
I hadn't heard of that convention, I certainly don't use it, but fair enough renamed to `liftOldestFirstF` and co. (My first version was in fact called `liftOldestFirstM` but then I realized it just needed a `Functor` constraint, not a `Monad` constraint, and I wasn't sure what name to use anymore :).